src/runtime/runtime-basic.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright 2020 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AutoPromptType} from '../api/basic-subscriptions';
+import {ConfiguredRuntime} from './runtime';
+
+const RUNTIME_BASIC_PROP = 'SWG_BASIC';
+
+/**
+ * Reference to the runtime, for testing.
+ * @private {!BasicRuntime}
+ */
+let basicRuntimeInstance_;
+
+/**
+ * Returns runtime for testing if available. Throws if the runtime is not
+ * initialized yet.
+ * @visibleForTesting
+ * @return {!BasicRuntime}
+ */
+export function getBasicRuntime() {
```

Comment:
rename this file as basic-runtime.js so it will be consistent with basicRuntime, basicSubscription, etc.

Reply:
Done.